examples/ppm_example.py

```diff
@@ -0,0 +1,43 @@
+from pylab import *
+from pyspecdata import *
+
+# make peaks on a frequency axis
+x = nddata(r_[0:8:49152j], "t2")
+x.set_units("t2", "s")
+
+# generate time-domain signal
+signal = 0 * x  # create an array of zeros that's the right shape
+SFO1 = 400.130438262389
+TMS_shift = 12.5
+for A, nu, R in [
+    (0.1, TMS_shift, 0.04),
```

Comment:
say that this represents my TMS peak -- how would I adjust my parameters to set this to 0 ppm?

Reply:
from a manual for Bruker 
![image](https://user-images.githubusercontent.com/31140790/136456561-f13b30dc-ad26-4657-9422-453233acf401.png)
-- I believe TMS is defined as the peak location where offset is 0, or where SFO1 = BF -- we cannot touch the BF, just the offset. It is still not extremely clear what happens. But to answer the question, that is you want this peak to represent TMS, wouldn't you just set TMS_shift = 0 ? For TMS, there would be no shift for it relative to its own peak, so a shift of 0 would just be the SFO1 which is consistent with what I am saying based off the Bruker manual

Reply:
this is not correct --> TMS is rarely if ever on resonance with the carrier peak.  Turns out that the parameter I'm asking about is `SR` --> definitely check it out to cross-check.  More on this in next reply.